sunpy/physics/transforms/differential_rotation.py

```diff
@@ -1,8 +1,7 @@
 import warnings
-import textwrap
-warnings.warn(textwrap.dedent("""\
-                  The module 'sunpy.physics.transforms.differential_rotation.py' is deprecated and
-                  will be removed in the future. Use 'sunpy.physics.differential_rotation.py' instead"""),
-              stacklevel=2)
+from sunpy.util.exceptions import SunpyDeprecationWarning
```

Comment:
@nabobalis When did we deprecate this? Does this need pulling for 1.0?

Reply:
Ok, so we deprecated these in 0.9, we should remove them. You want to do that here @ehsteve or seperately?